docs/getting_started/sharp-bits.md

```diff
@@ -1,79 +1,101 @@
 # The sharp bits 🔪
 
-In dynamiqs a syntax similar to that of QuTiP is provided, yet with distinct differences. This similarity can lead to a false sense of familiarity, resulting in inadvertent errors. Below, we highlight several common pitfalls that users may encounter when learning to use the library.
+This page highlight common pitfalls that users may encounter when learning to use dynamiqs.
 
-## Operating on operators
-### Adding scalars to operators
+In dynamiqs we use PyTorch tensors to represent quantum states and operators. A PyTorch tensor is very similar to a NumPy array, and most NumPy functions have a PyTorch equivalent. However, PyTorch tensors can be stored on GPU, and can be attached to a *computation graph* to compute gradients. This makes them very powerful, but also introduces some subtleties that you should be aware of.
 
-In QuTiP, adding a scalar to a `QObj` performs an implicit multiplication of the scalar with the identity matrix. Conversely, in dynamiqs, adding a scalar to a tensor performs an element-wise addition. To achieve the same result as in QuTiP, you must **explicitly multiply the scalar with the identity matrix**.
+Here's a short summary of the different sections for the fast-paced reader:
+
+- [**Main differences with QuTiP**](#main-differences-with-qutip)
+    - [**Adding a scalar to an operator**](#adding-a-scalar-to-an-operator) &nbsp; Use `A + 2 * dq.eye(n)` instead of `A + 2`.
+    - [**Multiplying two operators**](#multiplying-two-operators): use `A @ B` instead of `A * B`, and `torch.linalg.matrix_power(A, 4)` instead of `A**4`.
+    - [**Computing the adjoint**](#computing-the-adjoint): use `dq.dag(x)`, `x.mH` or `x.adjoint()` instead of `x.dag()`.
+- [**Use a NumPy function**](#use-a-numpy-function): use `x.numpy()` to convert your tensor to a NumPy array (don't do this if you use a GPU or compute gradients).
+- [**RuntimeError: element 0 of tensors does not require grad and does not have a grad_fn**](#runtimeerror-element-0-of-tensors-does-not-require-grad-and-does-not-have-a-grad_fn): the tensor on which you want to compute the gradient is not attached to the computation graph.
+- [**Using a for loop**](#using-a-for-loop): use batching rather than a `for` loop to simulate multiple Hamiltonians or initial states.
```

Comment:
Not a huge fan of this summary. I find it a bit cluttered. Plus, the tutorial is not so long that it's required I think.

I would suggest reducing its size, putting it inside an admonition (like you did for `Defining Hamiltonians` page, or removing it altogether.

Reply:
I agree, I removed it.